privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -502,14 +508,15 @@ def getUserList(self, searchDict):
         # do the filter depending on the searchDict
         filter = u"(&" + self.searchfilter
         for search_key in searchDict.keys():
+            # convert to unicode
+            searchDict[search_key] = to_unicode(searchDict[search_key])
```

Comment:
Hmm, we are modifying the passed dictionary here. Maybe we should add a hint to the docstring ("Caution: The argument ``searchDict`` is modified in-place!") or create a copy of it.